Reenable new Gallery perf test

New gallery perf test is failing due to Gradle
```
2020-05-01T09:50:05.019349: stdout: [+1611 ms] 
stdout:            FAILURE: Build failed with an exception.
stdout: 
stdout:            * Where:
stdout:            Build file '/Users/flutter/.pub-cache/hosted/pub.dartlang.org/shared_preferences_macos-0.0.1+6/android/build.gradle' line: 22
stdout: 
stdout:            * What went wrong:
stdout:            A problem occurred evaluating root project 'shared_preferences_macos'.
2020-05-01T09:50:05.019505: stdout:            > Failed to apply plugin [id 'com.android.library']
stdout:               > Minimum supported Gradle version is 5.4.1. Current version is 5.1.1. If using the gradle wrapper, try editing the distributionUrl in /Users/flutter/.pub-cache/hosted/pub.dartlang.org/shared_preferences_macos-0.0.1+6/android/gradle/wrapper/gradle-wrapper.properties to gradle-5.4.1-all.zip
```

We'll reenable it once we figure out how to fix this gradle issue.

--- a/dev/devicelab/lib/tasks/new_gallery.dart
+++ b/dev/devicelab/lib/tasks/new_gallery.dart
@@ -19,7 +19,7 @@ class NewGalleryPerfTest extends PerfTest {
     // Manually roll the new gallery version for now. If the new gallery repo
     // turns out to be updated frequently in the future, we can set up an auto
     // roller to update this version.
-    await getNewGallery('59489e5571ddf554fc62ef52e9b16f1fed291026', galleryDir);
+    await getNewGallery('c1682ac4b35dc239f023eaf0238a98910b3a2d6c', galleryDir);
     return await super.run();
   }
 

--- a/dev/devicelab/manifest.yaml
+++ b/dev/devicelab/manifest.yaml
@@ -784,6 +784,7 @@ tasks:
       Measures the performance of screen transitions in the new Flutter Gallery on Android.
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
+    flaky: true
 
   fast_scroll_large_images__memory:
     description: >

--- a/dev/devicelab/manifest.yaml
+++ b/dev/devicelab/manifest.yaml
@@ -779,12 +779,11 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["linux/android"]
 
-# TODO(https://github.com/flutter/flutter/issues/56135)
-#  new_gallery__transition_perf:
-#    description: >
-#      Measures the performance of screen transitions in the new Flutter Gallery on Android.
-#    stage: devicelab
-#    required_agent_capabilities: ["mac/android"]
+  new_gallery__transition_perf:
+    description: >
+      Measures the performance of screen transitions in the new Flutter Gallery on Android.
+    stage: devicelab
+    required_agent_capabilities: ["mac/android"]
 
   fast_scroll_large_images__memory:
     description: >

